e2e/pages/createClaim/claimantSolicitorServiceAddress.page.js

```diff
@@ -0,0 +1,30 @@
+const address = require('./../../fixtures/address.js');
+const postcodeLookup = require('./../../fragments/addressPostcodeLookup');
+
+const {I} = inject();
+
+module.exports = {
+
+  fields: {
+    applicantSolicitor1ServiceAddress_hasServiceAddress: {
```

Comment:
How about `applicantSolicitor1HasServiceAddress`? We shouldn't mix camel case with underscores in property names.